components/raftstore/src/router.rs

```diff
@@ -47,17 +48,13 @@ where
     }
 
     /// Sends RaftCmdRequest to local store.
-    fn send_command(&self, req: RaftCmdRequest, cb: Callback<EK::Snapshot>) -> RaftStoreResult<()> {
-        send_command_impl::<EK, _>(self, req, cb, None)
-    }
-
-    fn send_command_with_deadline(
+    fn send_command(
         &self,
         req: RaftCmdRequest,
         cb: Callback<EK::Snapshot>,
-        deadline: Deadline,
+        extra_opt: RaftCmdExtraOpt,
```

Comment:
Should be `RaftCmdExtraOpts` or `RaftCmdOptions`. There are several configurable options in it.

Reply:
Done!